<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />




<title>Curriculum vitae</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/flatly.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 60px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 65px;
  margin-top: -65px;
}

.section h2 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h3 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h4 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h5 {
  padding-top: 65px;
  margin-top: -65px;
}
.section h6 {
  padding-top: 65px;
  margin-top: -65px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Pietro Delfino</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About Me</a>
</li>
<li>
  <a href="workfolio.html">Workfolio</a>
</li>
<li>
  <a href="cv.html">CV</a>
</li>
<li>
  <a href="publications.html">Publications</a>
</li>
<li>
  <a href="contact.html">Contact</a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore"><em>Curriculum vitae</em></h1>

</div>


<p>Department of Biotechnology, University of Verona</p>
<p>Strada Le Grazie 15, Verona</p>
<p>pietro.delfino (at) univr (dot) it</p>
<p><a href="www.github.com/pietrod"></a></p>
<p>Born April 26, 1988 - Genova</p>
<div id="research-experience" class="section level4">
<h4>RESEARCH EXPERIENCE</h4>
<p><em>May-November 2015</em> <strong>Laimburg Research Center</strong> - Researcher in the lab of Applied Genomics and Molecular Biology - Advisor: Dr. Thomas Letschka</p>
<p><em>December 2013-January 2015</em> <strong>ETH Zürich</strong> - Research Assistant at the Institute of Agricultural Sciences - Advisor: Prof. Bruno Studer</p>
<p><em>November 2012-November 2013</em> <strong>Fondazione Edmund Mach</strong> - MSc Internship in the Department of Genomics - Advisor: Dr. Daniel J. Sargent</p>
</div>
<div id="education" class="section level4">
<h4>EDUCATION</h4>
<p><em>2015-present</em> <strong>University of Verona</strong> - PhD Biotechnology and Bioinformatics - Tutor: Prof. Diana Bellin Title: <em>‘Candidate-genes association study for grapevine veraison’</em></p>
<p><em>2010-2013</em> <strong>University of Bologna</strong> - MSc Evolutionary Biology, graduated <em>cum laude</em> - Supervisor: Dr. Riccardo Velasco</p>
<p><em>2007-2010</em> <strong>University of Genova</strong> - BSc Medical Biotechnology - Supervisor: Prof. Ranieri Cancedda</p>
</div>
<div id="teaching-experience" class="section level4">
<h4>TEACHING EXPERIENCE</h4>
<p><em>2015-2016</em> <strong>University of Verona</strong> - Course ‘Omics Sciences’: Laboratory of RNA-Seq Data Analysis</p>
</div>
<div id="honors-awards" class="section level4">
<h4>HONORS &amp; AWARDS</h4>
<p><em>June 2016</em> <strong>X International Symposium on Grapevine Physiology and Biotechnology</strong> - Poster award winner for the section <em>‘Advances in phenotyping, -omic technologies, large data’</em></p>
</div>
<div id="journals-reviewed-for" class="section level4">
<h4>JOURNALS REVIEWED FOR</h4>
<p><em>BMC Bioinformatics</em></p>
</div>
<div id="publications" class="section level4">
<h4>PUBLICATIONS</h4>
<p>Buti, M., Sargent, D. J., Mhelembe, K. G., <strong>Delfino, P.</strong>, Tobutt, K. R., &amp; Velasco, R. (2016). Genotyping-by-sequencing in an orphan plant species <em>Physocarpus opulifolius</em> helps identify the evolutionary origins of the genus <em>Prunus</em>. <em>BMC Research Notes</em>, 9, 268. <a href="http://doi.org/10.1186/s13104-016-2069-4" class="uri">http://doi.org/10.1186/s13104-016-2069-4</a></p>
</div>
<div id="skills" class="section level4">
<h4>SKILLS</h4>
<ul>
<li>Languages
<ul>
<li>English (C1)</li>
<li>German (A1.2)<br />
</li>
</ul></li>
<li>Computer, programming, informatics
<ul>
<li>Unix operating systems</li>
<li>Bash scripting (intermediate)</li>
<li>R for statistics, bioinformatics - <em>Bioconductor</em>, data visualization (upper intermediate)</li>
<li>R scripting (intermediate)</li>
<li>Python (basic)</li>
<li>Version control - <em>Git</em>, <em>Github</em></li>
<li>Reports, interactive tables, graphics and reports - <em>Shiny</em>, <em>Plotly</em><br />
</li>
</ul></li>
<li>Bioinformatics and computational biology
<ul>
<li>Whole genome sequencing (<em>Illumina</em>) data analysis: quality control, alignment, variant calling, filtering, SNP functional prediction</li>
<li>Microarray data analysis</li>
<li>RNA-Seq data analysis: quality control, traditional and alignment-free methods, differential expression, gene ontology enrichment, network and pathway analysis</li>
<li>Re-analysis of published omics data from GEO and SRA databases</li>
<li>Meta-analysis and integration of genetic, genomic and transcriptomic data</li>
<li>Population genetics and genomics (phylogenetic analysis, population structure)</li>
<li>Genetic map construction from molecular markers (SSR and SNP)<br />
</li>
</ul></li>
<li>Molecular biology
<ul>
<li>DNA and RNA purification from animal and plant tissues</li>
<li>PCR, qPCR, High Resolution Melting (HRM)</li>
<li>Fragment analysis (capillary electrophoresis)</li>
</ul></li>
</ul>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
